src/components/additional-information/styles.module.scss

```diff
@@ -0,0 +1,22 @@
+.additionalInformation {
+  position: relative;
+  padding: 8px 24px;
+  font-family: "Open Sans", sans-serif;
+  font-size: 16px;
+  font-style: italic;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #687481;
```

Comment:
@IliaSolovev  @RomanAchilov  can't we just introduce a `variables.scss` to avoid extra-work of replacing these values later?